pages/[metricType]/[metricId].tsx

```diff
@@ -0,0 +1,104 @@
+import { format } from 'date-fns'
+import ReactECharts from 'echarts-for-react'
+import { GraphQLClient } from 'graphql-request'
+import Head from 'next/head'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+
+import { DateRangePicker, Layout } from '@/components'
+import { CounterQuery, TimeSeriesQuery } from '@/graphql'
+import { buildCounterChartConfig, buildTimeSeriesChartConfig } from '@/utils'
+
+const client = new GraphQLClient(
+  process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT_US_EAST_2
+)
+
+export default function MetricVisualization() {
+  const [title, setTitle] = useState('')
+  const [description, setDescription] = useState('')
+  const [options, setOptions] = useState('' as any)
+  const [startDate, setStartDate] = useState(new Date('2021-01-01T10:00:00Z'))
```

Comment:
Currently, the counter visualization uses a hard-coded relative time range, so these dates are only used for the time series visualization.

I'll work on improving that functionality over the next month or so.